sysl2/sysl/importer/tests-grammar/go.gen.g

```diff
@@ -0,0 +1,57 @@
+goFile: PackageClause Comment? '\n' ImportDecl? '\n' TopLevelDecl+ '\n';
```

Comment:
This is not a useful "test". Tests need to test 1 thing and 1 thing only. Using this test input, when a test fails the developer will have to then go digging "Something broke" is not helpful. "Union types are broken" Is much more useful outcomes.

Remove this and the java one, add multiple TINY test input files to get the required 99% coverage.

At a minimum I'd expect the following tests:
1. Just a single rule with only a single string
1. 2 rules, one referring to the other
1. simple but with withe * and ? qualifiers
1. simple with the choices (and choices with qualifiers).

Reply:
I am not sure if the amount of effort involved justifies the payout.

Reply:
Please stop with this attitude! If these tests plus whatever else are needed to get to complete coverage takes more than 2 hours it almost certainly means the implementation is incredibly fragile. Test-after is extremely poor engineering. Test-driven design should be part of your standard work flow.

Reply:
Apologies for replying before my morning coffee, after looking back I realised my remark was far more aggressive than necessary.

I don't think there needs to be more than half a dozen tests to get full coverage for this code, and each test need only be a few lines of grammar. I think the effort then is very low regardless of the possible payout

Reply:
We developers are folks passionate about technology and solid code. so no worries mate!  I have parked this PR for now and starting working on gRPC, will fix this and others once I am through with that.